Library/Homebrew/cmd/install.rb

```diff
@@ -281,6 +281,33 @@ def run
           puts "You're on your own. Failures are expected so don't create any issues, please!"
         end
 
+        overwrite = if !args.overwrite? &&
+                       GitHub::Actions.env_set? &&
+                       ENV["HOMEBREW_GITHUB_ACTIONS_NO_INSTALL_OVERWRITE"].blank? &&
```

Comment:
Should we document this in `env_config.rb`?

```suggestion
                       ENV["HOMEBREW_NO_GITHUB_ACTIONS_INSTALL_OVERWRITE"].blank? &&
```

would be my preferred naming format, for consistency